### ISSUE TITLE
Track recent breaking changes in `exceptions.net*.json` files

### ISSUE THREAD

**Issue by dougbu** [aspnet/EntityFramework#7958]
This repo contains untracked breaking changes. To find and address them
1. Revert d30fd13ea
2. Run `.\build.cmd /t:Pack` in projects d30fd13ea touched or `.\build.cmd /t:Package` in the repo root
3. Update `exceptions.net45.json` and `exceptions.netcore.json` to track the changes
4. If not already announced, suggest filing issues at https://github.com/aspnet/Announcements

**Comment by dougbu** [comment]
In addition the following projects lack baselines. The details below are reformatted from a recent Universe build on the CI. (Of course, disabling API Check for a couple of the projects listed may be appropriate.)
```
[EntityFramework\src\EFCore.Relational.Design.Specification.Tests\EFCore.Relational.Design.Specification.Tests.csproj]
  No baseline file for net46 found in EFCore.Relational.Design.Specification.Tests.
  No baseline file for netstandard1.6 found in EFCore.Relational.Design.Specification.Tests.
[EntityFramework\src\EFCore.Relational.Specification.Tests\EFCore.Relational.Specification.Tests.csproj]
  No baseline file for netstandard1.3 found in EFCore.Relational.Specification.Tests.
[EntityFramework\src\EFCore.Specification.Tests\EFCore.Specification.Tests.csproj]
  No baseline file for net46 found in EFCore.Specification.Tests.
  No baseline file for netstandard1.3 found in EFCore.Specification.Tests.
[EntityFramework\src\EFCore.Sqlite\EFCore.Sqlite.csproj]
  No baseline file for netstandard1.3 found in EFCore.Sqlite.
[EntityFramework\src\EFCore.Tools.DotNet\EFCore.Tools.DotNet.csproj]
  No baseline file for netcoreapp1.0 found in EFCore.Tools.DotNet.
[EntityFramework\src\EFCore.Tools\EFCore.Tools.csproj]
  No baseline file for net46 found in EFCore.Tools.
  No baseline file for netcoreapp1.0 found in EFCore.Tools.
```

**Comment by dougbu** [comment]
Not positive (didn't check all package ids) but it appears these packages all shipped in 1.0.0. If that's right, baselines should be created in `release` or `rel/1.0.1.1` and checked into at least `dev`.

**Comment by smitpatel** [comment]
Specification.Tests projects are explicitly ignored. See https://github.com/aspnet/EntityFramework/issues/6996#issuecomment-276262616

Rest of the projects were added to the repo after we had initially created baselines. I will add baselines for them. Thanks for info.
